internal/pkg/describe/env.go

```diff
@@ -146,6 +150,27 @@ func (d *EnvDescriber) Version() (string, error) {
 	return metadata.Version, nil
 }
 
+// ServiceDiscoveryEndpoint returns the endpoint the environment was initialized with, if any. Otherwise,
+// it returns the legacy app.local endpoint.
+func (d *EnvDescriber) ServiceDiscoveryEndpoint() (string, error) {
+	p, err := d.Params()
+	if err != nil {
+		return "", fmt.Errorf("get params of environment %s in app %s: %w", d.env.Name, d.env.App, err)
+	}
+	for k, v := range p {
+		// Ignore non-svc discovery params
+		if k != cfnstack.EnvParamServiceDiscoveryEndpoint {
+			continue
+		}
+		// Stacks upgraded from legacy environments will have empty ServiceDiscoveryEndpoint params.
+		// Stacks created at 1.5.0 or later will have a non-empty value of `env.app.local`
+		if v != "" {
+			return v, nil
+		}
```

Comment:
Is this comment accurate now that the default is `{{AppName}}.local`?

Reply:
no, fixed.